common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -12063,7 +12071,7 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     /**
      * whether or not blocking common passwords is enabled
      *
-     * @since ZCS 8.9.0
+     * @since ZCS 9.1.0
```

Comment:
why are you changing here?

Reply:
I didn't change this at all, but i don't know why this change appears. it was changed by @chinmay15 in https://github.com/Zimbra/zm-mailbox/commit/d27f5920a6920b84e8a81a81df1f45ea4658710a
But he seems forgot to run `ant generate-getters`

Reply:
not sure, how did i miss that..